packages/examples/main/src/langium/main.ts

```diff
@@ -149,6 +121,29 @@ const run = async () => {
     reader.onClose(() => languageClient.stop());
 };
 
-setup()
+await initServices({
+    enableThemeService: true,
+    enableTextmateService: true,
+    enableModelEditorService: true,
+    modelEditorServiceConfig: {
+        useDefaultFunction: true
+    },
+    enableConfigurationService: true,
+    configurationServiceConfig: {
+        defaultWorkspaceUri: '/tmp'
+    },
+    enableKeybindingsService: false,
+    enableLanguagesService: true,
+    enableAudioCueService: true,
+    enableDebugService: true,
+    enableDialogService: true,
+    enableNotificationService: true,
+    enablePreferencesService: true,
+    userServices: {
+        ...getKeybindingsServiceOverride()
```

Comment:
Why enabling it by hands?

Reply:
Just to show it is possible to doing it both ways. A comment would not hurt here.

Reply:
Added a comment, see commit below ⬇️